### PR TITLE
Add contact and group counters to address books

### LINKED ISSUE #3320: Add a subline to each of the address books, explain what an address book is
### Is your feature request related to a problem? Please describe.

It would be helpful if:
* [ ] Add a subline to each of the address books saying e.g. "250 contacts, 10 groups" to indicate what address books are
* [ ] Optional: add text explaining what an address book is. Address books are collections of contacts. You can organize contacts into more than one address book and share individual address books with other accounts.

### Describe the solution you'd like

_No response_

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by DJdisco07**
Fixes #3320 

image

### CODE REVIEW

File: src/components/AppNavigation/Settings/SettingsAddressbook.vue [hunk 1]
```diff
@@ -24,11 +24,19 @@
...
+			<div class="addressbook__content">
+				<!-- addressbook name -->
+				<span class="addressbook__name" :title="addressbook.displayName">
+					{{ addressbook.displayName }}
+				</span>
 
+				<!-- counters -->
+				<div class="addressbook__count-wrapper">
+					<span class="addressbook__count">{{ groupsCount }} contact(s)</span>
+					<span class="addressbook__count">- {{ contactsCount }} group(s)</span>
```
Comment (ChristophWurst) [inline src/components/AppNavigation/Settings/SettingsAddressbook.vue:36]:
```suggestion
					<span class="addressbook__count">{{ n('contacts', '%n contact', '%n contacts', contactsCount) }} contact(s)</span>
					<span class="addressbook__count">- {{ n('contacts', '%n group', '%n groups', contactsCount) }} group(s)</span>
```

First parameter is the app name
Second parameter is the singular translation
Third parameter is the plural translation
Fourth parameter is the number of elements

https://docs.nextcloud.com/server/latest/developer_manual/basics/front-end/l10n.html?highlight=translation#javascript

groupsCount and contactsCount were mixed up

**Review by ChristophWurst — CHANGES_REQUESTED** [review]
Please add a screenshot to the pull request description showing the visual changes (example PR: https://github.com/nextcloud/contacts/pull/3346)

**Review by ChristophWurst — APPROVED** [review]
Tested and works

run `npm run lint:fix` to auto-fix indentation and other formatting issues

**Review by ChristophWurst — APPROVED** [review]
Code looks good and the feature works as expected

Please squash your four commits into one so it forms an *atomic* change

Thank you :pray:

**Comment by ChristophWurst** [comment]
I think the rebase went sideways. Could you hard-reset your local branch to 6851011f1ac249e0dad3a017981788178c116da2 and redo the rebase & squash to latest origin/main?

File: src/components/AppNavigation/Settings/SettingsAddressbook.vue [hunk 2]
```diff
@@ -209,6 +217,25 @@ export default {
...
+
+		groups() {
+		const allGroups = this.contacts
+			.flatMap(contact => contact.vCard.getAllProperties('categories').map(prop => prop.getFirstValue()));
+		// Deduplicate
+		return [...new Set(allGroups)];
+		},
+
+		contactsCount() {
+      	return this.contacts.length;
```
Comment (ChristophWurst) [inline src/components/AppNavigation/Settings/SettingsAddressbook.vue:233]: indentation looks off now. are tabs and spaces mixed up?

Comment (DJdisco07) [reply src/components/AppNavigation/Settings/SettingsAddressbook.vue]: The changes of 6851011 were disregarded when resetting. I made the reset again by committing them.

**Comment by DJdisco07** [comment]
> I think the rebase went sideways. Could you hard-reset your local branch to [6851011](https://github.com/nextcloud/contacts/commit/6851011f1ac249e0dad3a017981788178c116da2) and redo the rebase & squash to latest origin/main?

**Pull request merged**